### PR TITLE
Execute hanadb_exporter as a daemon

### PR DESCRIPTION
**Pull request by arbulu89**
Execute hanadb_exporter as a daemon.
And hanadb_exporter installation is suggested (even though it is installed in the salt execution itself)

### CODE REVIEW

File: hana/exporter.sls [hunk 1]
```diff
@@ -5,7 +5,8 @@
 {% if node.host == host and node.exporter is defined %}
 
 {% set instance = '{:0>2}'.format(node.instance) %}
-{% set config_file = '/etc/hanadb_exporter/config_{}_{}.json'.format(node.sid, instance) %}
+{% set daemon_instance = '{}_{}'.format(node.sid, instance) %}
```
Comment (MalloZup) [inline hana/exporter.sls]: I personally prefer `{0} {1}` it make the code imho more implicit. But is just a personal style, ok with both.

File: hana/exporter.sls [hunk 2]
```diff
@@ -22,15 +23,11 @@ configure_exporter:
...
-stop_exporter:
-  process.absent:
-    - name: hanadb_exporter -c {{ config_file }} -m /etc/hanadb_exporter/metrics.json
-    - require:
-        - configure_exporter
-
 start_exporter:
-  cmd.run:
-    - name: nohup hanadb_exporter -c {{ config_file }} -m /etc/hanadb_exporter/metrics.json &>/dev/null &
+  service.running:
```
Comment (MalloZup) [inline hana/exporter.sls]: in theory we could also `retry` the service start .. but imho we can just avoid for moment  this :grin:

File: hana/exporter.sls (hunk 1 above)
Comment (arbulu89) [reply hana/exporter.sls]: @MalloZup Do you mean `{0}_{1}`? As this extension will be used to match with a file name, we cannot have spaces.

Comment (MalloZup) [reply hana/exporter.sls]: @arbulu89 I mean the format thing in python. you can write `{0}` or `{}` they are identic. 

but is same.. :grin:

Comment (MalloZup) [reply hana/exporter.sls]: I'm resolving now

File: hana/exporter.sls (hunk 2 above)
Comment (MalloZup) [reply hana/exporter.sls]: resolving

**Pull request merged**